### ISSUE TITLE
NuScenes Replication of Results

### ISSUE THREAD

**Issue by YilmazKadir** [valeoai/ALSO#5]
Than you for publishing your work and code. I tried to replicate the results for SemanticKITTI and NuScenes datasets for semantic segmentation using MinkUNet. For SemanticKITTI I get 62.7 mIoU(scratch) which is perfect but for NuScenes I get 71.3(scratch) which is a bit more than expected(70.2). I only changed train split to train and val split to val from the original config file of nuscenes. I attach the output file so maybe you can help. Thanks in advance.

[output.txt](https://github.com/valeoai/ALSO/files/10282699/output.txt)

**Comment by aboulch** [comment]
Hello,
Did you use the `eval.py` to compute the final scores?
If you report only the scores for the training script, there may be a small discrepancy.
For fast evaluation in the training script, we do not reproject on the original point cloud (the score is given on the 16k input points).
The correct way to evaluate is given in the `eval.py` script.

**Comment by YilmazKadir** [comment]
Hi,
Thank you for the quick response. I did as you say and get a very similar result (71.28) again. This model is trained on whole train dataset(28k scenes). I also trained a model on "parametrizing"(24k scenes) and that achieves 70.1 mIoU. Are the reported values for NuScenes validation trained on whole train dataset or "parametrizing" dataset?

**Comment by aboulch** [comment]
Checking the logs you are right: training on parametrizing and evaluation on official validation set.
I will make the changes in the code.
Thanks a lot for pointing out.

**Comment by aboulch** [comment]
Hello,
We updated the paper on arxiv: on nuScenes, provided scores are done finetuning with the complete training set.
